janusgraph-core/src/main/java/org/janusgraph/util/system/NetworkUtil.java

```diff
@@ -20,17 +20,7 @@
 
 public class NetworkUtil {
     public static String getLoopbackAddress() {
-        // InetAddress.getLoopbackAddress() is @since 1.7
-        //
-        // Aside from that, getLoopbackAddress() seems preferable to
-        // InetAddress.getByName("localhost") since the former doesn't seem to
-        // require the local resolver to be sane.
-        //return InetAddress.getLoopbackAddress().getHostAddress();
-        try {
-            return InetAddress.getByName("localhost").getHostAddress();
-        } catch (UnknownHostException e) {
-            throw new RuntimeException(e);
-        }
+        return InetAddress.getLoopbackAddress().getHostAddress();
```

Comment:
Does getLoopbackAddress never throw UnknownHostException?

Reply:
It does not throw any exceptions declared or appear to otherwise.